package.json

```diff
@@ -11,7 +11,7 @@
     "build:reactjs-todo": "nx run reactjs-todo:build",
     "build:angular-todo": "nx run angular-todo:build",
     "changeset": "changeset",
-    "clean": "shx rm -rf ./{coverage,bundles,dist,docs,lib,lib-esm,node_modules,tests,tmp,todos,users}/ ./samples/*/node_modules/ ./samples/_static/js/*/ ./samples/todo-api/{todos,users} ./packages/*/{bundles,lib,lib-esm}/ && git clean -fX -e \"!.env*\"",
+    "clean": "shx rm -rf ./{coverage,bundles,dist,docs,lib,lib-esm,node_modules,tests,tmp,todos,users}/ ./samples/*/node_modules/ ./samples/_static/js/*/ ./samples/todo-api/{todos,users} ./packages/*/{bundles,lib,lib-esm}/ && git clean -fX -e \"!.env*,nx-cloud.env\"",
```

Comment:
Should the addition be prefixed with `!`? I can't remember the reason for it, so I looked it up:

> An optional prefix "!" which negates the pattern; any matching file excluded by a previous pattern will become included again. 

Not super helpful, but I think since it's ignored in the `.gitignore`, I think we need the `!` to negate the it? Ugh, not the most intuitive, so maybe I'm wrong.

Reply:
I tested this locally and it seems to keep the file when running `npm run clean`
![Screenshot 2023-09-27 at 1 31 04 PM](https://github.com/ForgeRock/forgerock-javascript-sdk/assets/18267769/d1f8559c-1ab3-4f3f-ab1b-39a928012b21)

Reply:
I refreshed the tree before this Screenshot also